util/opentelemetry-util-http/src/opentelemetry/util/http/__init__.py

```diff
@@ -162,10 +162,13 @@ def normalise_response_header_name(header: str) -> str:
 def get_custom_headers(env_var: str) -> List[str]:
     custom_headers = environ.get(env_var, [])
     if custom_headers:
-        custom_headers = [
-            custom_headers.strip()
-            for custom_headers in custom_headers.split(",")
-        ]
+        if custom_headers.strip().lower() == "all":
```

Comment:
Even if the odds are low, I suggest not having this `all` option. Since the regex already does the job, I would add a note on how to capture all the headers in the doc.

Reply:
Removed.